pkgs/development/ocaml-modules/patricia-tree/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  findlib,
+  mdx,
+  qcheck-core,
+  re,
+  camlp-streams,
+  csexp,
+  astring,
+  ocaml-version,
+  fmt,
+  logs,
+  ppx_inline_test,
+}:
+
+buildDunePackage rec {
+  pname = "patricia-tree";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "codex-semantics-library";
+    repo = "patricia-tree";
+    tag = "v${version}";
+    hash = "sha256-lpmU0KhsyIHxPBiw38ssA7XFEMsRvOT03MByoJG88Xs=";
+  };
+
+  strictDeps = true;
+
```

Comment:
```suggestion
```

This line is redundant.

Reply:
Why? I have it in all the other new packages.